boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad_iron.dts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54h20dk_nrf54h20_cpurad.dts"
+#include "nrf54h20dk_nrf54h20-ipc_conf_iron.dtsi"
+#include "nrf54h20dk_nrf54h20-memory_map_iron.dtsi"
+
+/delete-node/ &cpusec_cpuapp_ipc;
```

Comment:
This is already done in https://github.com/zephyrproject-rtos/zephyr/blob/76e1fc7713a4a3f2b50c497afddec0364a34c10b/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad.dts#L15

Reply:
fixed in https://github.com/zephyrproject-rtos/zephyr/pull/89333